src/groovy/org/pih/warehouse/importer/ProductAssociationExcelImporter.groovy

```diff
@@ -38,6 +39,7 @@ class ProductAssociationExcelImporter extends AbstractExcelImporter {
             "associatedProduct.name"        : ([expectedType: ExcelImportUtils.PROPERTY_TYPE_STRING, defaultValue: null]),
             "quantity"                      : ([expectedType: ExcelImportUtils.PROPERTY_TYPE_INT, defaultValue: null]),
             "comments"                      : ([expectedType: ExcelImportUtils.PROPERTY_TYPE_STRING, defaultValue: null]),
+            "hasMutualAssociation"          : ([expectedType: ExcelImportUtils.PROPERTY_TYPE_STRING, defaultValue: "no"]),
```

Comment:
@drodzewicz won't this defaultValue evaluate to `true` if it is a string?

Reply:
yes it will, I accidentally rollbacked changes for validation where I check `if(params.hasMutualAssociation == "yes")`  
I updated the PR